src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -48,11 +48,11 @@
   -->
 
   <xsl:template name="pickfont-sans">
-    <xsl:text>Arial,sans-serif</xsl:text>
+    <xsl:text>Liberation Sans,Arial,sans-serif</xsl:text>
   </xsl:template>
 
   <xsl:template name="pickfont-serif">
-    <xsl:text>Georgia,serif</xsl:text>
+    <xsl:text>DejaVu Serif,Liberation Serif,serif</xsl:text>
```

Comment:
> NotoSerif and OpenSans

I do not use them myself but on the screen they look Ok. The Apache license is also Ok. Have you seem them printed together? I am working on a bigger document and will give them a try.

> That's actually already true, but this change will cause the output to change for people who currently rely on this script (in a different, inconsistent way).

Yes, if we want to improve some the output should change ;)

We could see it this way: On Windows there are 2 types of users: 
a) those who do not care about fonts and licenses or those who are not able to install fonts:
-> As long as the MS-core fonts are listed as replacement fonts nothing will change for them.
b) those who care and who install open fonts.
-> They appreciate tools that do not default to proprietary (Microsoft) fonts. And, they are technically skilled enough to change our defaults. if ever they do not like them.
RESULT:
A good choice of open fonts as default (with MS fonts as fall-backs) suits everybody.